app/views/posts/show.html.erb

```diff
@@ -1,2 +1,30 @@
-<h1>Posts</h1>
-<p> Shows info of a single post</p>
\ No newline at end of file
+<div class='center2'>
+    <div class='border'>
+        <div class='flex jstSbt'>
+            Post by <%= @post.author.name %>
+            <div>
+                Comments : <%= @post.comment_counter %>,
+                Likes : <%= @post.likes_counter %>
```

Comment:
-  [ ]  Please add a margin to the top of posts to show page
![image](https://user-images.githubusercontent.com/74744787/218282993-47154cc4-360c-4d76-805a-07bfecd02d0e.png)